### PR TITLE
Change up build dialogue

### PR DESCRIPTION
**Pull request by RDIL**
Adds placeholder, different config options, etc.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "prop-types": "*",
     "query-string": "*",
     "react": "*",
-    "react-ace": "*",
+    "react-ace": "^7.0.2",
```
Comment (fkorotkov) [inline package.json]: revert

**Comment by RDIL** [comment]
Okay react doesn't want to start a dev server :/

**Comment by RDIL** [comment]
Never mind, but I'm not able to sign in without being redirected to the real Cirrus CI.

**Comment by RDIL** [comment]
Ok, here is the difference
before
Screenshot 2019-08-06 at 11 19 21 AM


after
Screenshot 2019-08-06 at 11 20 16 AM

yeah I spelled highlighting wrong im half asleep

**Comment by fkorotkov** [comment]
What's the reasoning behind switching the theme? `github` one seems more appropriate since at the moment GitHub is the only supported platform.

**Comment by RDIL** [comment]
Mainly visibility, line numbers can more easily seen and the pink looks pretty

**Comment by fkorotkov** [comment]
It does look pretty but not consistent with the GitHub UI though. 🤷‍♂️There must be another option to make line number more visible.

Please also revert changes to `schema.graphql`

**Pull request merged**